src/Config/Config.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+export interface OktaConfig {
+  baseUrl: string;
+  issuer: string;
+  clientId: string;
+  redirectUri: string;
+}
+
+export interface ServiceConfig {
+  measureService: {
+    baseUrl: string;
+  };
+  elmTranslationService: {
+    baseUrl: string;
+  };
+}
```

Comment:
There is another ServiceConfig in `madie-cql-library` that exposes a `cqlLibraryService` config.

Reply:
Added accordingly.
We will want to retrieve config from util in a later ticket